csm/conf/csm_cleanup.py

```diff
@@ -79,38 +86,64 @@ def remove_old_indexes(es, arg_d, arg_n, arg_e):
         else:
             Log.debug("Nothing to remove")
 
-def process_stats(args):
-    # Pass arguments to worker function
-    es = esCleanup(const.CSM_CLEANUP_LOG_FILE, const.CSM_LOG_PATH)
-    remove_old_indexes(es, args.d, args.n, args.e)
 
-def process_auditlogs(args):
+def clean_indexes(es,no_of_days,host_port):
+    for index,field in index_field_map.items():
+        Log.debug(f"Removing data for old index:{index} for {no_of_days} days.")
+        es.remove_old_data_from_indexes(no_of_days, host_port, [index], field)
+    remove_old_indexes(es, no_of_days, host_port, args.emulate)
+
+def get_du_data():
+    cmd = "sudo du -BM /var/log/elasticsearch/"
```

Comment:
From Line no 97-101 and 106-110 the code is getting repeated. You can have a common function for this and pass command as input and have the csm_data as return value.

Reply:
`get_du_data` and `get_df_data` returns different data. Also parsing of data is in different way. That is why different functions are ccreated

Reply:
You can keep `get_du_data` and `get_df_data` functions as it is. But you can have a new function for the common code that you can call in both the functions.

Reply:
Comment addressed. Please review